tutorials/getting-started.adoc

```diff
@@ -2,6 +2,18 @@
 
 == Installation
 
+=== System Requirements
+
+The system requirements for the SMART COSMOS DevKit are as follows:
+
+minimum::
+* 2 CPUs (CPU cores)
+* 5 GB Memory
+
+recommended::
+* 4 or more CPUs (CPU cores)
+* 16 GB Memory
+
```

Comment:
Do you have any idea what the disk space requirements are, @jbauerrfid ?

Docker states 3 GB. [MariaDB](https://mariadb.com/kb/en/mariadb/mariadb-hardware-requirements/) doesn't state anything specific (and I think that's a good idea):
> MariaDB doesn't have minimal hardware requirements per se, or if there was any, it would correspond to pre-2000 machines which don't exist anymore. So it mostly depends on what you intend to do with it. Tarball is approximately 400MB, by my last count. Packages are much less than that.

I particularly like the "it mostly depends on what you intend to do with it", it's probably the exact same thing for the DevKit. However, what we could do is sum up the size of the containers, even though I don't think that would be an accurate number for disk space requirements due to the layered images.